test/testkube/testkube-test-crs.yaml

```diff
@@ -64,6 +64,15 @@ spec:
       branch: ci_prod
       path: test/ginkgo-e2e
   executionRequest:
+    variables:
+      AZURE_TENANT_ID:
+        name: AZURE_TENANT_ID
+        value: "72f988bf-86f1-41af-91ab-2d7cd011db47"
```

Comment:
lets NOT expose this. Can we use the pipeline variable to pass the value of AZURE_TENANT_ID?